reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
Personally I don't like very much these two fields names even if they are X3DOM names because they are not very descriptive.
If the ``fields`` field can only contain Uniform nodes, then I would rather call it ``uniformVariables`` or something like this.
But as we discussed on Tuesday then it is fine to use them.